### ISSUE TITLE
Add text to icon buttons

### ISSUE THREAD

**Issue by simevidas** [HubPress/hubpress.io#389]
I accidentally published a post because I thought the icon meant “Save to GitHub”.

***

Before:

![before](https://cloud.githubusercontent.com/assets/716405/16172754/d7ab6282-358e-11e6-88c5-245de6a0ee5f.png)

After:

![after](https://cloud.githubusercontent.com/assets/716405/16172755/d9ffd892-358e-11e6-8fcd-8080427bca17.png)

**Comment by jaredmorgs** [comment]
I like the idea of Textual labels, but perhaps keep them as hover-text, or long tap in Android.

If you were using this on a Tablet in Portrait, those text labels would be cut off. Particularly with release 0.5.1 now having an extra icon in the blog editing view.

I've noticed that if you long tap on the icons in Android Marshmallow, Google Now On Tap displays the original file name of the icon pack, not the actual function of the button.

@anthonny should fix that behaviour so it meets Accessibility guidelines.

**Comment by anthonny** [comment]
I like this idea too.
There a big rework to do with the UI, in the first the UI was handcrafted and in the current version i use material-ui.

But for me it is not beautiful, there is not any identity instead of the logo and the orange color :)

**Comment by simevidas** [comment]
If you make people hover despite this much empty space in the bar (on desktop), people might get annoyed; there is responsive design (CSS media queries) to show/hide element parts based on available space 😋

Another ambiguity:

![foo](https://cloud.githubusercontent.com/assets/716405/16177866/c421a686-3638-11e6-8de5-fdb824c53ab2.png)

**Comment by anthonny** [comment]
> there is responsive design (CSS media queries)

Yes, i use MQ to adapt the view on mobile only. Like i said i did not invest a lot of time on the design of HubPress 😄 
If you have some skills in UX/UI and have time, feel free to make some proposal 👍 

I agree with you the pen is not the best icon to say "new post", a plus should be better :D

**Comment by simevidas** [comment]
@anthonny Where could I check out the code that generates those UI buttons? The [app.js file](https://github.com/HubPress/hubpress.io/blob/master/hubpress/scripts/app.js) is a minified bundle 😅

**Comment by anthonny** [comment]
Everything is here https://github.com/HubPress/dev.hubpress.io/tree/development
It's a React app using Redux and Material-ui

**Comment by anthonny** [comment]
Feel free to join our slack http://hubpressio-slack.herokuapp.com if you have any question

**Comment by jaredmorgs** [comment]
It is now a vue2 app. 😁 

I still think this is a valid issue. The textual lables under each icon would be better in mobile browsers because you can't hover over the button.

**Comment by anthonny** [comment]
I agree with you @jaredmorgs it is still a valid issue
